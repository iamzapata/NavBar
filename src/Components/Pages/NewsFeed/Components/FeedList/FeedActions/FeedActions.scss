@import "styles/variables.scss";

.FeedActions {
  button {
    border: 0;
  }
  .icon {
    color: $color-gray-content-text;
  }
}
