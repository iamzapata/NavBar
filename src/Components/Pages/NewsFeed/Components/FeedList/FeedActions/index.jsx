import React from "react"
import "./FeedActions.scss"

const FeedActions = () => (
  <div className="FeedActions evel-right">
    <div className="level-item">
      <button className="button" type="button">
        <span className="icon">
          <i className="fa fa-ellipsis-h" aria-hidden="true" />
        </span>
      </button>
    </div>
  </div>
)

export default FeedActions
