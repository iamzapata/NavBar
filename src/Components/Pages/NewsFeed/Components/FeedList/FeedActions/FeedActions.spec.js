import React from "react"
import { render, cleanup } from "react-testing-library"
import FeedActions from "./index"

describe("<FeedActions />", () => {
  afterEach(cleanup)
  it("Should render self", () => {
    const { container } = render(<FeedActions />)
    expect(container.firstChild).toBeTruthy()
    expect(container.firstChild).toMatchSnapshot()
  })
})
