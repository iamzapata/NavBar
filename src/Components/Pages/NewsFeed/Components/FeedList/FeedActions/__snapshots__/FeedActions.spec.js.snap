// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FeedActions /> Should render self 1`] = `
<div
  class="FeedActions evel-right"
>
  <div
    class="level-item"
  >
    <button
      class="button"
      type="button"
    >
      <span
        class="icon"
      >
        <i
          aria-hidden="true"
          class="fa fa-ellipsis-h"
        />
      </span>
    </button>
  </div>
</div>
`;
